mock/etc/bash_completion.d/mock

```diff
@@ -2,7 +2,12 @@
 
 _mock_root()
 {
-    COMPREPLY+=( $( compgen -W "$( command ls ${1:-/etc/mock} 2>/dev/null | \
+    if [ -z "$1" ]; then
+        CFGDIRS=(/etc/mock, "$HOME/.config/mock/")
+    else
+        CFGDIRS=($1)
+    fi
+    COMPREPLY+=( $( compgen -W "$( command ls \"${CFGDIRS[@]}\" 2>/dev/null | \
```

Comment:
I'm not sure `ls` is good option here.  If you list one directory, only the files are on the output;  but if you have two directories, you have the format:
```
<DIR>:
file1 file2 file3

<DIR2>:
....
```

Reply:
maybe `$( for dir in "${CFGDIRS[@]}"; do ls "$dir"; done )`;  it would remove the `command` builtin, but I'm not sure it is needed ..